auto_round/compressors/config.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2025 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Union
+
+import torch
+
+
+class ExtraConfig:
+    """Class for extra or legacy configs."""
+
+    config_type: str = "base"
+
+    def __init__(
+        self,
+        amp: bool = True,
+        device: Union[int, str, torch.dtype] = None,
+        dynamic_max_gap: int = -1,
+        disable_deterministic_algorithms: bool = True,
+        disable_opt_rtn: bool = True,
+        enable_alg_ext: bool = False,
+        enable_deterministic_algorithms: bool = False,
+        enable_minmax_tuning: bool = True,
+        enable_norm_bias_tuning: bool = False,
+        enable_quanted_input: bool = True,
+        fp_layers: str = None,
```

Comment:
better remove fp_layers, if layer_config and fp_layers both set the same layer but with different bits, I guess fp_layers will override the layer_config. Please make sure the behavior is correct.